cgic.c

```diff
@@ -1873,6 +1873,7 @@ void cgiHeaderCookieSetString(char *name, char *value, int secondsToLive,
 }
 
 void cgiHeaderLocation(char *redirectUrl) {
+	fprintf(cgiOut, "Status: 303 Redirecting\r\n"); // neccesary on some servers, eg litespeed
```

Comment:
Understand, however as this is used quite often I would like to make sure you tested this. Can you try a few different web servers and clients?